CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 * For MySQL users, a massive scalability improvement via batched DB writing of internal metadata events. Note that one must add `rewriteBatchedStatements=true` to their JDBC URL in their config in order to take advantage of this
 * In some cases the SFS backend, used for Local, SGE, etc., coerced `WdlFile` to `WdlString` by using `.toUri`. This
 resulted in strings prepended with `file:///path/to/file`. Now absolute file paths will not contain the uri scheme.
+* Added docker hash lookup. Docker images without an explicit digest value will disqualify the task for call caching. See https://github.com/broadinstitute/cromwell#call-caching-docker-tags for more details
```

Comment:
I was just trying to view it from the lens of someone reading changelog. My first question is going to be "so? what does that mean?" - and i tend to want instant answers w/ changelog-y stuff. See your point though, so if you can think of way that hits at both sides that's also cool